src/DependencyInjection/Admin/TaggedAdminInterface.php

```diff
@@ -91,6 +91,11 @@ public function getLabel();
      */
 //    public function showMosaicButton(bool $isShown): void;
 
+    /**
+     * @return array<string, array<string, mixed>>
+     */
+    public function getListModes();
```

Comment:
why this method should be in this interface?

Reply:
It's tricky but it's related to the method `showMosaicButton()`.

`showMosaicButton` is kinda the setter of the property `listModes `, and `getListModes` the getter.